pkg/api/v1/types.go

```diff
@@ -1241,6 +1243,10 @@ type PodSpec struct {
 	ImagePullSecrets []LocalObjectReference `json:"imagePullSecrets,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 }
 
+// PodSecurityContext holds pod-level security attributes and common container settings.
+type PodSecurityContext struct {
```

Comment:
Does that mean you're agreeing with the feedback and have not update this PR or arguing with the feedback?  I can't tell... :)